transactions/switchboard/transfer_tokens.cdc

```diff
@@ -0,0 +1,41 @@
+// This transaction is a template for a transaction that
+// could be used by anyone to send tokens to another account
+// through a switchboard, as long as they have set up their
+// switchboard and have add the proper capability to it
+//
+// The withdraw amount and the account from getAccount
+// would be the parameters to the transaction
+
+import FungibleToken from "./../../contracts/FungibleToken.cdc"
+import FungibleTokenSwitchboard from "./../../contracts/FungibleTokenSwitchboard.cdc"
+import ExampleToken from "./../../contracts/ExampleToken.cdc"
+
+transaction(to: Address, amount: UFix64) {
```

Comment:
```suggestion
import ExampleToken from "./../../contracts/ExampleToken.cdc"

transaction(to: Address, amount: UFix64, receiverPath: PublicPath) {
```

instead of explicitly using the switchboard contract here for this normal transfer, can you just have the user pass in an argument that is the switchboard public path and just borrow the receiver from there?